modules/markup/markdown/meta.go

```diff
@@ -25,15 +25,16 @@ func isYAMLSeparator(line string) bool {
 // and returns the frontmatter metadata separated from the markdown content
 func ExtractMetadata(contents string, out interface{}) (string, error) {
```

Comment:
AFAIU The yaml front matter must be the first thing in the file.

Therefore line[0] must be a line SEP and there must be another SEP later on
Then yaml starts at line[1] and continues until there is a SEP.

Thus this should be: check line 0, if SEP, find next SEP line and take lines [1:SEPline] to give to yaml. And the content is lines[SEPline+1:]

No need to keep two seps around.

Reply:
I suppose so. I guess the only difference is this would allow someone to have a blank line as the first line.  
Not sure why/when that would happen, but it could.

Do you want me to require the first sep being line 1? ~~Line 0 if you think in slices~~

Reply:
Dunno - have you seen a spec for yaml front matter? How does yuin meta handle this?

Reply:
It looks like it does enforce the first line being a separator. I can change it.

Reply:
Well, it now enforces a first separator. 
The bad news is the func is longer now. 

It occurred to me that `idx+1` is probably not a great idea in a slice of unknown length, e.g. a template with no body.

So now it checks for remaining lines before setting the body. This way someone could make an issue template with only frontmatter if they wanted.

Reply:
Go is very kind.

```go
a := "0123456789"
fmt.Println(a[9:], a[10:])
```
https://play.golang.org/p/1s393Y2bknc

Doesn't panic.

`a[10]` and `a[11:]` do panic, but a slice starting at `len(a)` doesn't.

Reply:
I see. Okay, I've changed it back to the one-line then. Tests for an empty body still passed :+1: